### PR TITLE
feat: [#229] Pay cycle calendar + 12-month projection

### LINKED ISSUE #229: [EPIC] Dashboard redesign Part 1: Pay Cycle Calendar + 12-month projection
## Goal
Redesign the dashboard's main content area around two new visual components based on the prototype at `thoughts/Can-Eye-Budget/Can I Budget.html`:

1. **Pay Cycle Calendar** — replaces "Recent Activity"
2. **12-Month Projection Chart** — new section at the bottom

## Scope (Part 1)
- ✅ New Pay Cycle Calendar Livewire widget
- ✅ New 12-Month Projection service + Livewire widget
- ❌ Out of scope: Verdict Hero, Money Triad restyle, sidebar polish (Part 2)

## Architectural Decisions
- New Livewire component `app/Livewire/Dashboard/PayCycleCalendar.php` (does NOT modify existing full-page `CalendarView`)
- Full prototype parity for calendar (14-day grid, pips, net per day, click-to-detail)
- ApexCharts v5.10.4 (already installed, currently unused)
- No caching for v1 — projection computed live
- Live computation strategy: planned-only expenses + pay-cycle-smoothed income

## Subtasks
- [x] #230 — Replace Recent Activity with Pay Cycle Calendar
- [x] #231 — Add 12-month projection chart

## Reusable infrastructure
- `User::currentPayCycleBounds()`, `daysUntilNextPay()`, `hasPayCycleConfigured()`
- `PlannedTransaction::occurrencesBetween()`, `excludingTransfers()` scope
- `Transaction::current()`, `excludingTransfers()` scopes
- CIB primitives: `x-cib.tx-row`, `x-cib.sec-head`, `x-cib.spark`, `x-cib.empty-state`

## Risks
- `wire:ignore` required on ApexCharts container (Livewire morphdom tears down chart otherwise)
- `MoneyCast::format` and JS formatter must match exactly
- `User::currentPayCycleBounds()` auto-fast-forwards past `next_pay_date` — assert against bounds in tests
- DST + timezone: use `CarbonImmutable::today()`; ensure `app.timezone` is set

## Reference
- Prototype: `thoughts/Can-Eye-Budget/Can I Budget.html`

### LINKED ISSUE #230: Replace dashboard "Recent Activity" with Pay Cycle Calendar widget
Parent EPIC: #229

> **⚠️ Bundle notice**
> This subtask was implemented **alongside #231** in the same branch (`feature/230-replace-dashboard-recent-activity-with-pay-cycle-calendar-widget`) and shipped as a single PR: **#232**.
>
> Both subtasks share two files (`resources/views/livewire/dashboard.blade.php` and `resources/css/app.css`), so splitting them into separate branches/PRs after the fact would have caused merge churn for negligible review benefit. The PR description in #232 lists each issue's deliverables separately. **Closing #230 will close #231 simultaneously** when #232 merges (both have `Closes #N` entries in the PR body).
>
> See **[## Verification of #231](#verification-of-231)** below for the full audit confirming #231 acceptance criteria are also met by this branch.

## What
Replace the "Recent Activity" section (lines 38-53 of `resources/views/livewire/dashboard.blade.php`) with a 14-day pay-cycle calendar grid showing transaction pips and net change per day. Match the prototype at `thoughts/Can-Eye-Budget/Can I Budget.html` (search `CalendarView` component, `.cyc-*` CSS classes).

## Files to create
- [x] `app/Livewire/Dashboard/PayCycleCalendar.php`
- [x] `app/Livewire/Dashboard/Data/PayCycleDayData.php`
- [x] `app/Livewire/Dashboard/Data/PayCyclePip.php`
- [x] `resources/views/livewire/dashboard/pay-cycle-calendar.blade.php`
- [x] `resources/views/components/cib/day-net.blade.php`
- [x] `tests/Feature/Livewire/Dashboard/PayCycleCalendarTest.php`

## Files to modify
- [x] `resources/views/livewire/dashboard.blade.php` (recent-activity section deleted, widget inserted in same grid column)
- [x] `resources/css/app.css` (`.cyc-*` rules ported into `@layer components`, reusing CIB tokens)
- [x] `tests/Feature/Livewire/DashboardTest.php` (replaced recent-activity assertion with `assertSeeLivewire('dashboard.pay-cycle-calendar')`)

## Component shape (delivered)
- [x] `cycleOffset` (int, 0=current, ±1=prev/next), `selectedDate` (Y-m-d, nullable)
- [x] Public actions: `previousCycle()`, `nextCycle()`, `goToCurrentCycle()`, `selectDay($iso)`
- [x] Computed: `bounds()`, `days()`, `selectedDay()`, `totals()`, `isCurrentCycle()`, `header()`
- [x] Cycle shifting: `match $user->pay_frequency` → days/months (weekly/fortnightly/monthly)

## Reuse confirmed
- [x] `User::currentPayCycleBounds()` for bounds
- [x] `Transaction::current()->excludingTransfers()` for posted txns
- [x] `PlannedTransaction::excludingTransfers()` + `occurrencesBetween()` for planned
- [x] Mirrors `CalendarView::calendarData()` query pattern (one query per source, grouped in PHP)

## Acceptance criteria
- [x] Recent Activity removed; widget rendered in same grid column
- [x] 14 days for fortnightly users, 7 for weekly, ~30 for monthly (all three covered by tests)
- [x] Cycle start = PAID, cycle end = PAYDAY badges
- [x] Today highlighted with TODAY pill
- [x] Up to 3 pips per day (income/planned/expense colored), `+N more` for overflow
- [x] Net change per day formatted with `MoneyCast::format`
- [x] Click a day → detail panel below grid lists that day's transactions
- [x] Prev/next cycle navigation works; `goToCurrentCycle()` resets
- [x] Empty-state when `!hasPayCycleConfigured()`, links to `pay-cycle.edit` route
- [x] Cycle spanning year boundary renders correctly (Dec 22 → Jan 5 test)
- [x] Transfer pairs excluded from pips
- [x] All Pest tests pass (`op test.filter PayCycleCalendar` → 19/19); `op ci` clean (Pint + PHPStan)

## Edge cases tested
- [x] More than 3 transactions on a day (`caps visible pips at 3 and surfaces +N more for overflow`)
- [x] Year-boundary cycle (`handles cycles spanning a year boundary`)
- [x] User with no pay cycle configured (`shows empty-state when pay cycle is not configured`)
- [x] Past/future cycles via `cycleOffset` (`previousCycle shifts...` × 3 frequencies)
- [x] Other users' transactions don't leak in (`isolates current user from other users transactions`)

---

#### Files created
- [x] `app/Services/Projection/MonthlyProjectionService.php` (123 lines)
- [x] `app/Services/Projection/ProjectedMonth.php` (32 lines)
- [x] `app/Services/Projection/ProjectedOneOff.php` (17 lines)
- [x] `app/Livewire/Dashboard/MonthlyProjection.php` (103 lines)
- [x] `resources/views/livewire/dashboard/monthly-projection.blade.php` (153 lines)
- [x] `tests/Feature/Services/Projection/MonthlyProjectionServiceTest.php` (261 lines, 15 tests)
- [x] `tests/Feature/Livewire/Dashboard/MonthlyProjectionTest.php` (73 lines, 4 tests)

#### Files modified
- [x] `resources/views/livewire/dashboard.blade.php` line 79: `<livewire:dashboard.monthly-projection lazy/>` appended full-width below the inner grid, inside `space-y-6` parent
- [x] `resources/css/app.css`: `.proj-*` / `.monthly-projection` / `.proj-warn` / `.proj-legend` / `.proj-empty` rules ported into `@layer components`

#### Service contract — verified at `MonthlyProjectionService.php:24`
```php
public function forUser(User $user, int $months = 12, bool $includeHistoricalBaseline = false): Collection
```
Matches the planned signature exactly. `$includeHistoricalBaseline` is reserved for v2 (Approach B, planned + historical baseline) and currently unused — explicit forward-compat hook.

#### Income strategy — `MonthlyProjectionService::smoothedMonthlyIncomeCents()` lines 104–114
- [x] Weekly: `intdiv($payAmount * 52, 12)` ✓
- [x] Fortnightly: `intdiv($payAmount * 26, 12)` ✓
- [x] Monthly: `$payAmount` ✓
- [x] Credit-direction `PlannedTransaction`s added to income (line 62-63)

#### Expense strategy (v1) — service lines 52–80
- [x] Sum of `PlannedTransaction::occurrencesBetween($monthStart, $monthEnd)` for active debits
- [x] Excludes transfers via `->excludingTransfers()` scope (line 37)

#### One-offs — service lines 70–79 + 116–122
- [x] `RecurrenceFrequency::DontRepeat` only
- [x] Debit direction only
- [x] Threshold = `min($200_cents, 0.25 × monthly_pay)` — i.e. qualifies if amount ≥ $200 OR ≥ 25% of pay (whichever is the looser threshold)
- [x] Surfaced as ApexCharts `annotations.points` with description tooltip (blade lines 118-139)

#### ApexCharts wiring — blade `monthly-projection.blade.php`
- [x] `wire:ignore` on container (line 43) — non-negotiable per plan, present
- [x] Mounted via Alpine `x-init` (line 147), not Livewire `$dispatch`
- [x] Cents sent to JS, divided by 100 in `formatMoney` (lines 50-57)
- [x] Chart `type: 'line'` umbrella with mixed series: 2 bars + 1 line (lines 64-78)
- [x] Colors: income `#2AAF46`, expense `#E5484D`, cumulative `#0E5E57` (line 79)
- [x] Risky bars colorized via `fill.colors` callback (lines 80-88) — non-risky months render at `#FBA3A6` (lighter red), risky months at `#E5484D` (full red)
- [x] One-offs as `annotations.points` (lines 118-139)

#### Acceptance criteria from #231
- [x] Service computes 12 months by default, supports custom count — test: `returns 12 months by default starting at the current month`, `returns the requested number of months when not 12`
- [x] Empty Collection when `!hasPayCycleConfigured()`; widget shows empty-state — service line 26-28; blade `@if ($this->projection->isEmpty()) <x-cib.empty-state ...>` line 4
- [x] Income smoothed across pay frequencies (all three) — tests: `income is smoothed for fortnightly pay using the 26/12 average`, `... weekly pay using the 52/12 average`, `income equals raw pay amount for monthly frequency`
- [x] Recurring plannings expanded across all months — test: `expands recurring planned transactions across all 12 months`
- [x] One-off plannings appear in matching month only — test: `surfaces DontRepeat debit planned transactions as one-offs in matching month`
- [x] Risky months flagged via `isRisky()` — `ProjectedMonth::isRisky()` method; test: `isRisky is true when expenses exceed income`
- [x] First risky month surfaced as warning banner above chart — `MonthlyProjection::firstRiskyMonth()` computed; blade `.proj-warn` block lines 31-40; test: `surfaces the closest risky month in a warning banner`
- [x] Cumulative net = running sum of `netCents` — service line 83 (`$cumulativeNetCents += $netCents`); test: `cumulative net is the running sum of monthly net`
- [x] User isolation — service line 35 filters `where('user_id', $user->id)`; test: `isolates one user from another users planned transactions`
- [x] Chart renders correctly with `wire:ignore`; tooltips formatted in dollars matching `MoneyCast::format` — `formatMoney(cents)` produces `$1,234.56` (en-AU `toLocaleString` with 2 decimals) which matches `MoneyCast::format`'s `$` + `number_format` + 2-pad behaviour
- [x] Lazy-loads — `<livewire:dashboard.monthly-projection lazy/>` (dashboard.blade.php line 79)
- [x] All Pest tests pass — `op test.filter MonthlyProjection` → 19/19; `op ci` Pint + PHPStan clean

#### Risks (#231 plan called out, all mitigated)
- `wire:ignore` mistake — present and verified (line 43)
- MoneyCast PHP vs JS format — both produce `$1,234.56`; en-AU locale matches PHP's `number_format` defaults
- `occurrencesBetween` `maxIterations = 1000` — fine for 12mo (planned `Everyday` would emit ~365 occurrences/yr × 12 = 4,380 over the full window, but the iteration cap is per-call-window, so each month-window call only sees ~31 max — plenty of headroom)
- DST: service uses `CarbonImmutable::today()->startOfMonth()` (line 40); no implicit UTC

---

✅ **Both #230 and #231 are fully implemented in this branch.** Final state to be confirmed once PR #232 merges.

### LINKED ISSUE #231: Add 12-month projection chart to dashboard
Parent EPIC: #229

## What
Add a 12-month forward projection chart at the bottom of the dashboard showing monthly income, expenses, cumulative buffer line, and one-off cost markers. Match the prototype at `thoughts/Can-Eye-Budget/Can I Budget.html` (search `Projection12Mo` component, `.proj-*` CSS classes). Implement with ApexCharts v5.10.4 (already installed).

## Files to create
- `app/Services/Projection/MonthlyProjectionService.php`
- `app/Services/Projection/ProjectedMonth.php`
- `app/Services/Projection/ProjectedOneOff.php`
- `app/Livewire/Dashboard/MonthlyProjection.php`
- `resources/views/livewire/dashboard/monthly-projection.blade.php`
- `tests/Feature/Services/Projection/MonthlyProjectionServiceTest.php`
- `tests/Feature/Livewire/Dashboard/MonthlyProjectionTest.php`

## Files to modify
- `resources/views/livewire/dashboard.blade.php` (append `<livewire:dashboard.monthly-projection lazy />` full-width below grid)
- `resources/css/app.css` (port `.proj-*` rules — warn banner, stat blocks, pin marker)

## Service contract
\`\`\`php
final readonly class MonthlyProjectionService {
    /** @return Collection<int, ProjectedMonth> */
    public function forUser(User \$user, int \$months = 12, bool \$includeHistoricalBaseline = false): Collection;
}
\`\`\`

## Income strategy
Pay-cycle smoothed: weekly × 52/12, fortnightly × 26/12, monthly × 1. Plus credit-direction `PlannedTransaction`s.

## Expense strategy (v1)
Planned-only: sum of `PlannedTransaction::occurrencesBetween(\$monthStart, \$monthEnd)` for debit plannings (excl. transfers). Constructor flag `\$includeHistoricalBaseline` reserved for v2.

## One-offs
`PlannedTransaction` with `RecurrenceFrequency::DontRepeat`, debit direction, threshold `>= \$200` OR `>= 25%` of monthly pay-amount. Pin marker on bar with description tooltip.

## ApexCharts mounting
- Mount via Alpine `x-init`, NOT Livewire `\$dispatch`
- **`wire:ignore` on chart container** — non-negotiable; without it Livewire morphdom destroys the chart on every action
- Send cents to JS, divide by 100 in formatter
- Chart type `line` with mixed series (2 bars + 1 line)
- Colors: income `#2AAF46`, expense `#E5484D`, cumulative `#0E5E57`
- Risky bars colorized via `fill.colors` callback
- One-offs as `annotations.points`

## Acceptance criteria
- [ ] Service computes 12 months by default, supports custom count
- [ ] Empty Collection when `!hasPayCycleConfigured()`; widget shows empty-state
- [ ] Income smoothed across pay frequencies (test all three)
- [ ] Recurring plannings expanded across all months
- [ ] One-off plannings appear in matching month only
- [ ] Risky months (`expense > income`) flagged via `isRisky()`
- [ ] First risky month surfaced as warning banner above chart
- [ ] Cumulative net = running sum of `netCents`
- [ ] User isolation: other users' plannings don't leak
- [ ] Chart renders correctly with `wire:ignore`; tooltips show formatted dollars matching `MoneyCast::format` output
- [ ] Lazy-loads (chart is below the fold)
- [ ] All Pest tests pass; `op ci` clean

## Risks
- `wire:ignore` mistake → chart blanks on any Livewire interaction
- MoneyCast PHP format vs JS format must match exactly
- `occurrencesBetween` `maxIterations = 1000` — fine for 12mo, watch for 24mo
- DST: use `CarbonImmutable`, never implicit UTC

### PR DESCRIPTION
**Pull request by robwilde**
## Summary

Delivers Part 1 of the dashboard redesign per [EPIC #229](https://github.com/robwilde/can-eye-budget-v2/issues/229), bundling both subtasks in one branch:

- **#230** — Replaces the dashboard's "Recent Activity" list with a 14-day Pay Cycle Calendar widget (transaction pips, planned occurrences, daily net, click-to-detail panel, prev/next/today navigation).
- **#231** — Adds a lazy-loaded 12-month forward projection chart at the bottom of the dashboard with pay-cycle-smoothed income, planned-only expenses, cumulative buffer line, risky-month flagging, and one-off cost markers.

## What changed

**New components**
- `App\Livewire\Dashboard\PayCycleCalendar` (+ `PayCycleDayData` / `PayCyclePip` DTOs)
- `App\Livewire\Dashboard\MonthlyProjection` (+ `App\Services\Projection\MonthlyProjectionService`, `ProjectedMonth`, `ProjectedOneOff` DTOs)
- `<x-cib.day-net>` Blade primitive for the colored ±$ net pill
- Two new Blade views under `resources/views/livewire/dashboard/`

**Modified**
- `resources/views/livewire/dashboard.blade.php` — replaced "Recent activity" with the calendar widget; appended `<livewire:dashboard.monthly-projection lazy/>` below the grid; outer wrapper bumped to `space-y-6`.
- `resources/css/app.css` — ported `.cyc-*` and `.proj-*` rules from the prototype into `@layer components`, reusing existing CIB tokens.
- `app/Livewire/Dashboard.php` — removed unused `recentTransactions` computed.
- `tests/Feature/Livewire/DashboardTest.php` — replaced obsolete recent-activity assertions with `assertSeeLivewire('dashboard.pay-cycle-calendar')`.

## Architectural decisions

- **ApexCharts mounting**: chart container has `wire:ignore` and mounts via Alpine `x-init`. Without `wire:ignore`, Livewire's morphdom tears down the SVG on every roundtrip — non-negotiable per the plan's risk list.
- **Income smoothing**: weekly × 52/12, fortnightly × 26/12, monthly × 1. Avoids the "3-pay-month" calendar artifact for fortnightly users (26 pays/yr ≠ 24).
- **Expenses (v1) — planned only**: deterministic, testable, consistent with `User::totalNeededUntilPayday()`. Constructor flag `$includeHistoricalBaseline = false` reserved for v2 (planned + 90-day historical-debit baseline) without rewrite.
- **One-off detection**: `PlannedTransaction` rows with `RecurrenceFrequency::DontRepeat`, debit direction, amount ≥ $200 OR ≥ 25% of monthly pay (whichever is lower). Surfaced as ApexCharts `annotations.points`.
- **Cycle rendering**: half-open `[bounds.start, bounds.end)` — exactly 14 days for fortnightly with `cycleStart` = PAID and the last cell = PAYDAY. The actual next payday (`bounds.end`) is the start of the next cycle.
- **No caching for v1** — projection computed live (one query for active planned txns + PHP loop). Escape hatch documented in plan if it ever becomes a hot path.
- **Component composition**: `MonthlyProjection` is a separate Livewire component (not a computed on `Dashboard`) so `wire:ignore` scopes cleanly and the chart can lazy-load below the fold.

## Test plan

- [x] `op test.filter PayCycleCalendar` — 19/19 passing
- [x] `op test.filter MonthlyProjection` — 19/19 passing (service + Livewire combined)
- [x] `op test.filter Dashboard` — 52/52 passing (no regressions)
- [x] `op test` (full suite) — 1,561 passed, 0 failures (4 pre-existing skips)
- [x] `op analyse` (PHPStan) — clean
- [x] `op lint.dirty` (Pint) — clean
- [ ] Manual smoke on `/dashboard` with seeded user — confirm calendar replaces recent activity, today highlighted, prev/next cycles, click-to-detail
- [ ] Manual smoke — confirm 12-month projection chart renders below grid, risky-month banner shows when applicable, tooltips formatted as dollars
- [ ] Empty-state check — fresh user without pay cycle: both widgets render their empty-state with link to `pay-cycle.edit`
- [ ] Year-boundary check — set `next_pay_date` to early Jan: calendar renders 14 days correctly across boundary
- [ ] Verify `wire:ignore` correctness — trigger any Livewire action elsewhere on the page after chart renders, confirm chart is intact

## Risks (called out in plan, mitigated here)

- `wire:ignore` on chart container — required, easy to forget.
- `MoneyCast::format` (PHP) vs JS formatter parity — both produce `$1,234.56` style output; tooltips match the rest of the dashboard.
- `User::currentPayCycleBounds()` auto-fast-forwards past `next_pay_date` — tests assert against the bounds, never `next_pay_date` directly.
- `PlannedTransaction::occurrencesBetween` has `maxIterations = 1000` — fine for 12 months, future 24-month projection with `Everyday` plannings would need attention.

## Closes

Closes #229
Closes #230
Closes #231

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: app/Livewire/Dashboard/PayCycleCalendar.php [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+            usort(
+                $dayPips,
+                static fn (PayCyclePip $a, PayCyclePip $b): int => $b->amount <=> $a->amount,
+            );
+
+            $hiddenCount = max(0, count($dayPips) - self::MAX_PIPS_PER_DAY);
+            $visiblePips = array_slice($dayPips, 0, self::MAX_PIPS_PER_DAY);
+
+            $days[] = new PayCycleDayData(
+                iso: $key,
```
[the remark this answers is not in the document]
Comment (robwilde) [reply app/Livewire/Dashboard/PayCycleCalendar.php:215]: ✅ **Fixed.** Confirmed real bug — the detail panel was reading the truncated `$day->pips` and silently dropping pips beyond the grid cap.

**Resolution:** stopped slicing in the component; `PayCycleDayData::$pips` now stores the full unsliced list. The grid view slices on render via `array_slice($day->pips, 0, PayCycleCalendar::MAX_PIPS_PER_DAY)`. The detail panel reads the full list, as does `selectedDay()['pips']`.

`MAX_PIPS_PER_DAY` constant visibility changed `private → public` so the Blade can reference it without hardcoding `3`.

**Regression test added:** `detail panel exposes all pips for a day even when more than the grid cap` asserts `selectedDay()['pips']` returns 5 when the day has 5 transactions. The repurposed `stores all pips and surfaces +N more hiddenCount for grid overflow` test confirms the DTO holds all 5 with `hiddenCount=2`.

20/20 PayCycleCalendar tests passing.

File: app/Livewire/Dashboard/PayCycleCalendar.php [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+
+    public function selectDay(string $iso): void
+    {
+        $parsed = CarbonImmutable::createFromFormat('Y-m-d', $iso);
+
+        $this->selectedDate = $parsed instanceof CarbonImmutable
+            ? $parsed->format('Y-m-d')
+            : null;
+
+        unset($this->selectedDay); // @phpstan-ignore property.notFound
```
[the remark this answers is not in the document]
Comment (robwilde) [reply app/Livewire/Dashboard/PayCycleCalendar.php:55]: ✅ **Fixed** — took the second option you suggested (stop persisting `isSelected` in the DTO and compare `$day->iso === $this->selectedDate` in the Blade).

**Why that route:** keeping `isSelected` on the DTO would have required `selectDay()` to bust *both* `$this->selectedDay` and `$this->days` caches on every click — and any future computed property that touches selection state would need to remember to re-bust. By deriving the active class at render time from the public property, the cache invalidation surface drops to zero on selection: Livewire's diff-based re-render handles the active class repaint, and `days()` stays cached across clicks (faster).

**Changes:**
- `PayCycleDayData` DTO no longer has an `isSelected` field
- `PayCycleCalendar::days()` no longer passes `isSelected` to the constructor
- Blade `@class([..., 'active' => $day->iso === $this->selectedDate])` derives live

The bug is now structurally impossible — there's no cached derived state to go stale.

File: resources/css/app.css [hunk 3]
```diff
@@ -685,6 +685,306 @@ select:focus[data-flux-control] {
...
         outline-offset: 2px;
     }
+
+    /* ---------- Pay-cycle calendar widget (Ticket #228) ---------- */
+
+    .pay-cycle-cal { display: flex; flex-direction: column; gap: 14px; }
+    .pay-cycle-cal .cyc-empty-link {
+        color: var(--color-cib-teal-500);
+        text-decoration: underline;
+        font: 700 13px/1 var(--font-sans);
```
[the remark this answers is not in the document]
Comment (robwilde) [reply resources/css/app.css]: ✅ **Fixed.** Comment renamed to `Ticket #230` for the calendar block. Leftover from when the redesign was originally drafted as a single ticket (#228) before it was split into the EPIC #229 + subtasks #230/#231 structure on the project board.

File: resources/css/app.css [hunk 4]
```diff
@@ -685,6 +685,306 @@ select:focus[data-flux-control] {
...
+        padding: 14px;
+        text-align: center;
+        color: var(--color-fg-3);
+        font: 700 13px/1.5 var(--font-sans);
+    }
+
+    /* ---------- 12-month projection widget (Ticket #228) ---------- */
+
+    .monthly-projection {
+        background: var(--color-bg-surface);
```
[the remark this answers is not in the document]
Comment (robwilde) [reply resources/css/app.css]: ✅ **Fixed.** Comment renamed to `Ticket #231` for the projection block. Same root cause as the calendar one — pre-split label.
